doc/flux_to_redux.md

```diff
@@ -0,0 +1,579 @@
+
+        ```dart
+        // An Influx Store
+        // Add the `InfluxStoreMixin` and use the Redux state class for the generic typing.
+        class ExampleInfluxStore extends flux.Store with InfluxStoreMixin<ReduxStateClass> {
+            ExampleFluxActions _actions;
+
+            // Point the `reduxReducer` getter at the Redux stete's reducer.
+            @override
+            get reduxReducer => lowLevelReduxReducer;
+
+            /// Use getters to point to state properties.
+            String get backgroundColor => state.backgroundColor;
+
+            ExampleInfluxStore(this._actions) {
+                state = ReduxStateClass.defaultState();
+                triggerOnActionV2(_actions.exampleFluxAction, 
+                    (_) => this.influxReducer(ExampleReduxAction()));
+            }
+        }
+
+        class ExampleReduxAction {}
+
+        class ExampleFluxActions {
+            final flux.Action<String> exampleFluxAction = flux.Action();
+        }
+        ```
+1. __Add adapter stores where necessary.__ Each store that will be Influx should be wrapped with a `FluxToReduxAdapterStore` to give Flux and Connected Flux access as well. In other words, if the store was updated to Redux directly or will only talk to Redux components, this step can be skipped.
+    
+    `FluxToReduxAdapterStore` takes in an Influx store and an actions instance. It should also receive the Influx store class and the Redux state class as generics.
+    ``` dart
+    FluxToReduxAdapterStore influxAdapterStore =
+        FluxToReduxAdapterStore<ExampleInfluxStore, ReduxStateClass>(exampleInfluxStoreInstance, exampleFluxActionsInstance);
+    ```
+1. __Create `Context` instances for the necessary stores.__ If a component will use multiple stores, a `Context` instance is required. The [OverReact Redux docs](./over_react_redux_documentation.md) have examples of having multiple stores, which describes the fundamentals of why this is necessary. In the end, it is most likely that the majority of `FluxToReduxAdapterStore` instances will need a `Context` instance. 
+1. __Wrap the component tree in a either a `ReduxProvider` or a `ReduxMultiProvider`.__ If there are multiple stores, a `ReduxMultiProvider` is more elegant and is encouraged. The store instances passed in should be the `FluxToReduxAdapterStore` instances, not the normal Flux store instances.
+1. __Refactor components.__ See the specific component type below for a reminder on which store instance is correct and any "gotchas" in the refactor. In general, remember that if a component talks to multiple stores, `composeHocs` can be used to simplify the connected factory declarations.
```

Comment:
I think we need at least a couple of examples of a flux component being converted to a redux one.

Reply:
I'll leave it up to you whether that gets put in its own dedicated section - and then just linked to from all the different places where we mention doing things like wrapping a component in a provider, etc. - or whether you add relevant examples in each place where mentioned.